packages/sanity/schemas/siteSettings.ts

```diff
@@ -0,0 +1,17 @@
+export default {
```

Comment:
now that were using sanity v3, could we just alter this slightly and make use of v3's `defineField` and `defineType` functions?

Reply:
ah yes, thanks for the reminder to use these